### ISSUE TITLE
build from source fails if GO111MODULE is not explicitly turned off

### ISSUE THREAD

**Issue by hnfgns** [ava-labs/gecko#152]
**Describe the bug**
build from source fails if GO111MODULE is not explicitly set to off

**To Reproduce**
Make sure GO111MODULE is not explicitly set to off and follow the instructions in README in verbatim
```
export GO111MODULE=""
go get -v -d github.com/ava-labs/gecko/...
cd $GOPATH/src/github.com/ava-labs/gecko

-bash: cd: .../go/src/github.com/ava-labs/gecko: No such file or directory
```

**Expected behavior**
`cd $GOPATH/src/github.com/ava-labs/gecko` should work

**Operating System**
osx

By submitting this issue I agree to the [Terms and Conditions](https://files.avalabs.org/legal/bug_hunt_q2_2020_terms.pdf) of the [Developer Accelerator Program](https://avalabs.org/dap).

**Comment by hnfgns** [comment]
explicitly disabling go modules seems to work `GO111MODULE=off go get -d -v github.com/ava-labs/gecko/...`

**Comment by danlaine** [comment]
Hey @hnfgns thanks for creating an issue.
We're switching to go modules shortly so this will be resolved then :)

**Comment by danlaine** [comment]
Resolved by #183
